orchestrator/devtools/scripts/migrate_pydantic_forms_imports.py

```diff
@@ -0,0 +1,61 @@
+"""Helper script to rewrite import statements in your orchestrator.
+
+Since types have been externalised in `pydantic_forms`, they were re-imported in `orchestrator.types` for backwards
+compatibility. These import statements have been removed, and therefore need to be updated in orchestrator
+implementations.
+"""
+
+import sys
+from pathlib import Path
+
+from shared import migrate, move_import
```

Comment:
This should be an absolute import, at least the script is failing on my machine when I'm trying to run it with 

```
python -m orchestrator.devtools.scripts.migrate_pydantic_forms_imports.py <dir>
```

Reply:
```
ModuleNotFoundError: No module named 'shared'
```